pkg/policy/groups/helpers_test.go

```diff
@@ -57,13 +57,22 @@ func (s *GroupsTestSuite) TestCorrectDerivativeName(c *C) {
 		derivativeCNP.ObjectMeta.Name,
 		Equals,
 		fmt.Sprintf("%s-togroups-%s", name, cnp.ObjectMeta.UID))
+
+	// Test clusterwide policy helper functions
+	ccnpName := "ccnp-test"
+	ccnp := getSamplePolicy(ccnpName, "")
+	derivativeCCNP, err := createDerivativeCNP(context.TODO(), ccnp, true)
```

Comment:
Naming here is confusing as `createDerivativeCNP()` would indicate a `CNP` being created, but then a `CCNP` is returned?

Reply:
That makes sense, I will update the name.